### PR TITLE
docs: refactor consumer-restriction.md

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update document under `docs/zh/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by SylviaBABY — APPROVED** [review]
LGTM, nice work!

**Review by SylviaBABY — APPROVED** [review]

File: docs/en/latest/plugins/consumer-restriction.md [hunk 1]
```diff
@@ -33,14 +32,14 @@ The `consumer-restriction` Plugin allows users to set access restrictions based
...
-| rejected_msg       | string        | False    |               |                                                                                           | Message returned when the request is rejected.                                 |
-| allowed_by_methods | array[object] | False    |               | ["GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS", "CONNECT", "TRACE", "PURGE"] | List of allowed HTTP methods for a Consumer.                                   |
+| Name               | Type          | Required | Default       | Valid values  | Description |
+|--------------------|---------------|----------|---------------|---------------|-------------|
+| type               | string        | False    | consumer_name | ["consumer_name", "service_id", "route_id"]  | Type of object to base the restriction on.  |
+| allowed_by_methods | array[object] | False    |               | ["GET", "POST", "PUT", "DELETE", "PATCH", "HEAD", "OPTIONS", "CONNECT", "TRACE", "PURGE"] | List of allowed HTTP methods for a Consumer. |
```
Comment (juzhiyuan) [inline docs/en/latest/plugins/consumer-restriction.md:38]: Hi 👋 May I know why this PR adjusts the order of the attributes? (The changes are not A-Z 😅)

Comment (lunarwhite) [reply docs/en/latest/plugins/consumer-restriction.md:38]: What do you mean by "A-Z"? 

I adjusted the order as I considered the priority of these three attributes. Please take a careful look at the `Description` of the `allowed_by_methods`, `whitelist` and `blacklist`.

Let me know if there are any concerns or suggestions. Thanks 🙏

Comment (juzhiyuan) [reply docs/en/latest/plugins/consumer-restriction.md:38]: But the `required` fields are moved down, right? Developers may ignore those `required` fields IMO.

Comment (lunarwhite) [reply docs/en/latest/plugins/consumer-restriction.md:38]: 👍 Good point, I didn't look at the `required` fields before.

Shall we just keep the original order?

Comment (juzhiyuan) [reply docs/en/latest/plugins/consumer-restriction.md:38]: Yes, I prefer keeping `Required` fields in the top :)

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**